src/Console/Shell.php

```diff
@@ -681,6 +681,8 @@ public function error($title, $message = null)
             $this->_io->err($message);
         }
         $this->_stop(1);
+
+        return 1;
```

Comment:
Exactly, and for `return $this->error()` to be valid in code itself (void does not allow that strictly speaking).